core/src/main/java/jenkins/model/item_category/ItemCategory.java

```diff
@@ -0,0 +1,112 @@
+package jenkins.model.item_category;
+
+import hudson.Extension;
+import hudson.ExtensionPoint;
+import hudson.model.ModelObject;
+import jenkins.model.Messages;
+
+/**
+ * A category for {@link hudson.model.Item}s.
+ *
+ * @since TODO
+ */
+public abstract class ItemCategory implements ModelObject, ExtensionPoint {
```

Comment:
@jglick As far I know, *A model object has a human readable name*. A `ItemCategory` has a name. Why not? If I remove this inheritance, Should it moves outside of `jenkins.model` package?